cloudify/workflows/workflow_context.py

```diff
@@ -1221,6 +1221,29 @@ def send_task(self, workflow_task, task):
 
         return result
 
+    def cancel_task(self, workflow_task, tenant, target, queue):
+        message = {
+            'service_task': {
+                'task_name': 'cancel-operation',
+                'kwargs': {'task_id': workflow_task.id}
+            }
+        }
+        if queue == MGMTWORKER_QUEUE:
+            vhost = broker_config.broker_vhost
+        else:
+            vhost = tenant['rabbitmq_vhost']
+
+        client = amqp_client.get_client(
+            amqp_user=tenant['rabbitmq_username'],
```

Comment:
good point. But it's the same for sending. We should likely change it? I think right now it only works because all users have privs to the `/` vhost anyway